x-pack/plugins/fleet/server/types/models/package_policy.ts

```diff
@@ -96,7 +96,8 @@ const PackagePolicyBaseSchema = {
   name: schema.string(),
   description: schema.maybe(schema.string()),
   namespace: schema.maybe(PackagePolicyNamespaceSchema),
-  policy_id: schema.string(),
+  policy_id: schema.maybe(schema.string()),
+  policy_ids: schema.maybe(schema.arrayOf(schema.string())),
```

Comment:
for BWC, added `policy_ids` as optional, and verifying in create package policy API that either `policy_id` or `policy_ids` is set

Reply:
It could be an improvement, currently I think empty strings would give an error when trying to find the agent policy down the line